examples/Vendored XCFramework Example/BananaLib/StaticLibrary/CoconutLib.xcframework/ios-arm64/Headers/CoconutLib.h

```diff
@@ -6,4 +6,4 @@ FOUNDATION_EXPORT double CoconutLibVersionNumber;
 //! Project version string for CoconutLib.
 FOUNDATION_EXPORT const unsigned char CoconutLibVersionString[];
 
-#import <CoconutLib/Coconut.h>
+#import "Coconut.h"
```

Comment:
do modular imports not work in a static library umbrella header?

Reply:
Not that I've been able to figure out. :/

Reply:
@Westacular I am curious what was the difficulty here? A new issue has been filed related to modular imports https://github.com/CocoaPods/CocoaPods/issues/10136 which led me back to this PR.